web/src/main/webapp/WEB-INF/config-security/openid-configuration-azure.json

```diff
@@ -0,0 +1,23 @@
+{"token_endpoint":"https://login.microsoftonline.com/87f91494-c0dc-493e-83c3-9226c111850a/oauth2/v2.0/token",
```

Comment:
This seems to be a sample file - if so then it should be renamed to indicate that it is a sample file.  And the uuid should probably be changed to generic ones like 00000000-0000-0000-0000-000000000000

Reply:
yes - this was an accident. will delete